cloud.foundry.cli/src/main/java/cloud/foundry/cli/services/CreateController.java

```diff
@@ -86,11 +130,21 @@ public Integer call() throws Exception {
             DefaultCloudFoundryOperations cfOperations = CfOperationsCreator.createCfOperations(loginOptions);
             ServicesOperations servicesOperations = new ServicesOperations(cfOperations);
 
-            for (Entry<String, ServiceBean> serviceEntry : serviceBeans.entrySet()) {
-                String serviceName = serviceEntry.getKey();
-                ServiceBean serviceBean = serviceEntry.getValue();
-                servicesOperations.create(serviceName, serviceBean);
-                Log.info("Service created:" , serviceName);
+            try {
+                // do so that authorization has taken place. else leads to authorization problems
+                // TODO: find better solution
+                cfOperations.getOrganizationId().block();
+                Flux.fromIterable(serviceBeans.entrySet())
+                        // delay that's used to to avoid race conditions and not deterministic errors
+                        // on the cloud foundry side
+                        // TODO: find better solution if possible
+                        .delayElements(Duration.ofSeconds(1))
+                        .flatMap(serviceEntry ->
+                            servicesOperations.create(serviceEntry.getKey(), serviceEntry.getValue()))
+                        .onErrorContinue(Log::warning)
+                        .blockLast();
+            } catch (RuntimeException e) {
+                throw new CreationException(e.getMessage());
             }
 
             return 0;
```

Comment:
We should return 1 if there were any problems during creation

Reply:
good point there

Reply:
This is done in the BaseControllers exception handling.

Reply:
I think this happens in the Base Controller class.

Reply:
The problem with the current implementation is, that we continue on errors, so we have no way to know if an error happend (and thus when to return 1 instead of 0). In the create/remove space developers commands the issue is solved by setting a boolean flag to true, if any error occurs during blocking.

Reply:
In the future we might have logic classes that would take care of signaling to the controller that an error occured. But for now we need a workaround for that.